tests/subsys/secure_storage/psa/its/prj.conf

```diff
@@ -0,0 +1,17 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACK_SIZE=3072
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_MBEDTLS=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_TIMER_RANDOM_GENERATOR=y
+CONFIG_MBEDTLS_ENTROPY_POLL_ZEPHYR=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
```

Comment:
I understand that the purpose of this code is to showcase that the internal storage proposed here is 100% compatible with the one provided by TF-M, but building for a TF-M plaform (ex: `nrf5340dk/nrf5340/cpuapp/ns` as listed in the `testcase.yaml`) gives a warning that
```
warning: MBEDTLS_PSA_CRYPTO_C (defined at modules/mbedtls/Kconfig.tls-generic:514,
modules/mbedtls/Kconfig.tls-generic:514) was assigned the value 'y' but got the value 'n'
```

Wouldn't it be nicer to split the case in `testcase.yaml` in order to have a configuration for TF-M and one for non-TF-M devices? So that both builds without warnings?

However I don't have a strong opinion on this, so if you don't agree just set this one as solved with a comment ;)

Reply:
Yeah, good point. I was aware of this, but as this single Kconfig fragment works for all board targets, I was like let's do it like this, I guess because of the previous discussion regarding this topic on the samples. And I thought that we wouldn't care about compilation warnings in tests, but it's true that it's not very clean.
I did what you suggested.